packages/block-editor/src/components/inspector-controls/groups.js

```diff
@@ -6,11 +6,15 @@ import { createSlotFill } from '@wordpress/components';
 const InspectorControlsDefault = createSlotFill( 'InspectorControls' );
 const InspectorControlsBlock = createSlotFill( 'InspectorControlsBlock' );
 const InspectorControlsAdvanced = createSlotFill( 'InspectorAdvancedControls' );
+const InspectorControlsDimensions = createSlotFill(
+	'InspectorDimensionsControls'
+);
 
 const groups = {
 	default: InspectorControlsDefault,
 	block: InspectorControlsBlock,
```

Comment:
I'm not sure "block" group for the inspector controls make sense.

Reply:
It comes from the parent PR that I opened. The reasoning is that `block` would contain all controls specific only to the particular block. Image is a great example here:

<img width="1296" alt="Screen Shot 2021-08-16 at 10 40 52" src="https://user-images.githubusercontent.com/699132/129536115-855306ba-2281-4353-b73e-31d707e75254.png">